menu.js

```diff
@@ -0,0 +1,10 @@
+const menu_button = document.getElementById("menu_btn");
+const close_menu = document.getElementById("close_menu");
+
+menu_button.addEventListener("click", () => {
+  mobile_menu.classList.add("show");
+});
+
+close_menu.addEventListener("click", () => {
+  mobile_menu.classList.remove("show");
+});
```

Comment:
- [ ] Hi @Roland-Ntwali, having made a great effort working on the mobile menu aspect of your portfolio project, I noticed there are still some things you need to make changes on. 
   - According to what we have in the Figma design after the mobile menu is displayed the content underneath it is `blur` but in your own design the contents are not `blur`. You can achieve blurring of the background content by making use of the CSS property called `backdrop-filter: blur(5px);`
   - Secondly, according to the project requirement when a user clicks on any of the mobile menu links it should direct me to the corresponding page in your project and the mobile menu should disappear, I noticed when I clicked the link, the mobile menu is not disappearing. Kindly fix this
   > N.B:  ![req](https://user-images.githubusercontent.com/78634841/215714833-f076b6a4-d501-48a7-b3b3-15ec3ca65fd1.png)

Figma design 👇 

![real-mob](https://user-images.githubusercontent.com/78634841/215605336-f4163c93-03b0-4f1d-9c38-571e5fb6c88b.png)

Your design 👇 

![rola](https://user-images.githubusercontent.com/78634841/215714021-202f8e81-a2da-444c-939d-5b681e0e0d5b.png)